api_draft.txt

```diff
@@ -0,0 +1,128 @@
+
+################################################################################
+Arimaa Game Server JSON API
+################################################################################
+
+#TODO - see todos below
+
+All requests and responses are in JSON.
+
+Shared types:
+<Message> = {"message":<String>}
+<Error> = {"error":<String>}
+<GameID> = <Int>   #Identifier for a game
```

Comment:
Maybe we should have a legacy game ID (for the purposes of being able to import the current game database), but otherwise use a hash. It's probably nice for users to be able to look up games by their current arimaa.com id, and then a hash actually becomes preferable since then it won't clash with that id. Thoughts?